chalice/deploy/deployer.py

```diff
@@ -229,6 +251,22 @@ def deploy(self, config, chalice_stage_name=DEFAULT_STAGE_NAME):
             chalice_stage_name: deployed_values
         }
 
+    def _get_chalice_deployment_error(self, error):
+        # type: (Any) -> ChaliceDeploymentError
+        suggestion = None
+        where = None
+        if isinstance(error, DeploymentPackageTooLargeError):
```

Comment:
Let's move this out into the `ChaliceDeploymentError` class.  We should want the deployer to only be about deploying resources and move the logic of helpful error messages out to a separate class.